libs/application-generic/src/services/types.ts

```diff
@@ -11,8 +11,15 @@ export interface IFeatureFlagContext {
   environmentId: EnvironmentId;
   organizationId: OrganizationId;
   userId: UserId;
+  [attribute: string]: unknown;
 }
 
+export type FeatureFlagFullContext = {
```

Comment:
Consider using a simpler type:

```suggestion
export type FeatureFlagContext = {
  scope: 'environment' | 'organization' | 'user',
  id: string,
  [key: string]: {}
```

Reply:
done